roles/docker/tasks/main.yml

```diff
@@ -24,5 +24,15 @@
   shell: systemctl daemon-reload && systemctl stop docker && systemctl start docker-tcp.socket && systemctl start docker
   when: docker_env_conf_result.changed or docker_tcp_conf_result.changed or docker_svc_conf_result.changed
 
+#NOTE: Without this step, pip crashes after installing docker-py/docker-compose
+#      Move to appropriate role later
+- name: upgrade pip
```

Comment:
This is actually pretty important for docker-compose use, but I have a 
patch for it that needs to be submitted to contiv/ansible still. if we 
end up removing this patch d-c may not work without it.

On 24 Dec 2015, at 17:17, Madhav Puri wrote:

>> @@ -24,5 +24,15 @@
>> shell: systemctl daemon-reload && systemctl stop docker && systemctl 
>> start docker-tcp.socket && systemctl start docker
>> when: docker_env_conf_result.changed or 
>> docker_tcp_conf_result.changed or docker_svc_conf_result.changed
>>
>> +#NOTE: Without this step, pip crashes after installing 
>> docker-py/docker-compose
>> +#      Move to appropriate role later
>> +- name: upgrade pip
>
> both these steps can go away if we don't use `docker` module as we 
> discussed yesterday.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/12/files#r48428468